src/components/menu-drawer/menu-drawer.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <v-navigation-drawer v-model="drawer" app clipped>
+  <v-navigation-drawer v-model="drawer" app clipped :mini-variant.sync="mini"
```

Comment:
Please format this line, use one line per parameter

`
<v-navigation-drawer
  :mini-variant.sync="mini"
  app
  clipped
  permanent
  v-model="drawer"
>
`

Reply:
done